docs/cookbook.md

```diff
@@ -460,6 +461,26 @@ implicit class RichSFuture[A](val f: SFuture[A]) extends AnyVal {
 }
 ```
 
+### Server Sent Events
+
+Finch offers support for Server Sent Events through the `finch-sse` sub-project.
+Server Sent Events are represented as `AsyncStream`s:
+```scala
+import io.finch.sse.ServerSentEvent
+import ServerSentEvent._
+
+def stream[A](sse: EventStream[A]): Output[EventStream[A]] = {
```

Comment:
What do you think about writing an example that streams current local time, every second? I'm happy to alter this example in the upcoming commit if you don't feel like doing it.